### DISCUSSION TITLE
"Only this visible" on Schematic

### DISCUSSION THREAD

**Discussion by F-Burning** [opentoonz/opentoonz#6457] [Ideas]
An option to only see the selected nodes (literally the nodes) to work with them without distractions in the schematic would be very helpful. This option don't affect to the viewer, only in the Schematic panel to improve the workflow.

**Comment by RodneyBaker** [comment]
I'm adding a workaround tag to this report because...

Use of shortcut key with 'Show this Only' works when selecting column nodes in the Schematic.

![image](https://github.com/opentoonz/opentoonz/assets/2576432/49737e49-50b5-4c95-ae49-5868ac445343)

Here in this example I've assigned Ctrl + T to "Show this Only" (think:  Control "This") and it works as expected to hide all columns except the one selected.  Selecting another node and using the shortcut then Shows only that column.
We would likely want to also assign a shortcut key to "Show All" to get all columns to show again.
I tested with use of Ctrl + Shift +A (thinking A for "All" as this is currently unassigned.

**Comment by RodneyBaker** [comment]
Recommendation:
Community should create a custom panel with desired shortcut keys assigned to buttons.
This to assist as a graphical menu that can be displayed while working in the Schematic.
Buttons assigned can be used to show currently selected columns as well as other useful options.
This can then be further customized by the user as desired.

From this use of custom panels developers can determine which are best integrated more fully via hardcoding into source.

Include this custom panel in distribution of Opentoonz.

This approach has several benefits to include freeing up shortcut keys, providing a graphical set of options extending current UI as well as demonstrating to users how custom panels can address their needs.

Follow up:
It's not clear that a custom panel will work as desired.
Issues include shortcuts that don't have useful icons, shortcuts that don't work where focus is lost, etc.  
As such this is an area for exporation more than a potential remedy.

For example, the letters of the shortcuts aren't particularly instructive but do work:

<img width="228" height="135" alt="Image" src="https://github.com/user-attachments/assets/dd1c5ee0-fc22-43e9-82d2-5c5d4ce329d2" />

Still it's better to have those letters than nothing.
